### ISSUE TITLE
using customInsertMenu now hides default components

### ISSUE THREAD

**Issue by cj** [BuilderIO/builder#167]
Using `customInsertMenu` did not use to hide ![](https://p192.p3.n0.cdn.getcloudapp.com/items/Z4uLmALl/Image%202020-05-09%20at%2012.10.44%20AM.png?v=e95d58b3bb823bc38ab23e296f742bf6)

Now when it is enabled via `Builder.set({customInsertMenu: true})` it completely removes it.

"@builder.io/react": "^1.1.18"

**Comment by steve8708** [comment]
Hey @cj - this has always been the intended behavior for `customInsertMenu` https://github.com/BuilderIO/builder/blob/master/examples/react-design-system/src/builder-settings.js#L24

Note that you can still register menus with `customInsertMenu` off _or_ on. Registering insert menus without making it fully custom just adds your menus and keeps the default blocks https://github.com/BuilderIO/builder/blob/master/examples/react-design-system/src/builder-settings.js#L32:L32

Also note that this can be a bit different using components only mode and various combinations of advanced options https://www.builder.io/c/docs/guides/components-only-mode

In short it's in your control what insert menus show, if the default blocks show or not, etc. If you can let us know what your desired insert menu would contain  (default blocks or not, custom menus or not, etc) we can point you in the right direction of what settings youll want to use
